### ISSUE TITLE
Training is not converged on syn-balanced and enzymes datasets

### ISSUE THREAD

**Issue by yyygou** [snap-stanford/neural-subgraph-learning-GNN#36]
Hi, it appears that I am encountering an issue while training a model on the syn-balanced and enzymes datasets.

In both cases, the training process seems to be running indefinitely, and the outputs are as follows:

Validation. Epoch 87. Acc: 0.5000. P: 0.5000. R: 1.0000. AUROC: 0.5000. AP: 0.5000.
TN: 0. FP: 2048. FN: 0. TP: 2048
Saving ckpt/model.pt
600it [00:00, 958.17it/s]. Training acc: 0.5000 000
600it [00:00, 980.59it/s] Batch 88394. Loss: 3.2000. Training acc: 0.5000
Batch 88999. Loss: 3.2000. Training acc: 0.5000

Everything remains unchanged except for the batch number.

Is there something wrong with the settings or configuration?

**Comment by fyulingi** [comment]
Hello, I'm having some environment configuration issues and cannot run the training codes. Could you please share your python version and your package version? Thx a lot!

**Comment by yyygou** [comment]
> Hello, I'm having some environment configuration issues and cannot run the training codes. Could you please share your python version and your package version? Thx a lot!

hi, I am using Python 3.6 in windows platform, and installed the packages in the following order:
1. numpy-1.17.1
2. matplotlib
3. networkx
3.1 deepsnap-0.1.2
4. torch
5. scipy-1.0.1
6. pandas
7. seaborn
8. scikit_learn
9. torch-cluster-1.5.3
10. torch_scatter-2.0.3
11. torch-sparse-0.6.0
12. torch-spline-conv-1.2.0
13. pytest-runner-5.3.2
14. numba-0.53.1
15. torch-geometric-1.4.3
16. test_tube == 0.7.5
17. tqdm-4.43.0

if the version number is missing, pls use the suggested version number given by the requirements.txt

**Comment by YiKangOY** [comment]
I also encountered the same problem, but some times the ACC will stuck at some other number, like 0.33 or 0.667 @yyygou May I ask have you resolved this issue?

**Comment by Jcorrieri** [comment]
For me, this was due to incompatibility with my graphics card (this version of Pytorch did not support my CUDA software), so I had to install the CPU version of Pytorch to see the model learn, though painfully slow.
